src/js/login/components/BetaDropdown.jsx

```diff
@@ -23,18 +23,12 @@ class BetaDropdown extends React.Component {
   }
 
   checkLink = (event) => {
-    if (event.target.tagName.toLowerCase() === 'a' && event.target.pathname === '/' && event.which === LEFT_CLICK) {
-      event.preventDefault();
-      event.stopPropagation();
-
-      this.redirectToDashboard();
+    const target = event.target;
+    if (target.tagName.toLowerCase() === 'a' && target.pathname === '/' && event.which === LEFT_CLICK) {
+      target.href = '/dashboard-beta';
     }
   }
```

Comment:
Even though it's mutating the event.target, this seems like a really solid way to do the redirect. What do you think?

Reply:
Changing to the `click` event solved the problem already (with redirects not working in higher environments) but I still think this is worth updating as well